### ISSUE TITLE
plugin/ecs: support autoscaling

### ISSUE THREAD

**Issue by richstokes** [hashicorp/waypoint#2162]
It would be great if Waypoint was able to configure autoscaling targets for applications.

**Comment by evanphx** [comment]
That would a great feature @richstokes! Reviewing the ECS docs on autoscaling, it seems like it's self-contained to ECS, so Waypoint just lacks the ECS specific configuration block to configured this. Does that seem right to you?

**Comment by richstokes** [comment]
Yep. You basically give ECS a metric to track (e.g. CPU/memoy load, SQS queue size) and it will autoscale a services' count based on that. It's all internal to ECS.
